crates/tuono/src/watch.rs

```diff
@@ -74,6 +77,14 @@ fn build_react_ssr_src() -> Job {
     .0
 }
 
+fn ssr_reload_needed(file_name: Option<&OsStr>, file_path: Cow<str>) -> bool {
+    file_name
+        .map(|f| f.to_string_lossy().starts_with(".env"))
+        .unwrap_or(false)
+        || file_path.ends_with("sx")
+        || file_path.ends_with("mdx")
+}
```

Comment:
Can we have this function taking only `path.0` as a single parameter? It is weird to me having it taking two arguments that are basically the same